client/reducers/index.js

```diff
@@ -4,17 +4,23 @@ import currentOrder from './currentOrder';
 import user from './user';
 import error from './error';
 import searchInput from './search'
+import account from './account'
 
 export default combineReducers({
   products: allProductsReducer,
   currentOrder,
   user,
   error,
+  account,
   searchInput
 })
 
 export * from './allProducts'
 export * from './currentOrder'
 export * from './user'
 export * from './error'
+<<<<<<< HEAD
```

Comment:
make sure to delete merging code

Reply:
It's not showing up in my file right now.  This might have been fixed already.